static/js/reactions.js

```diff
@@ -364,9 +364,6 @@ exports.view.remove_reaction = function (opts) {
     // the title/count and, if the user is the current user, turn off the
```

Comment:
Minor but please update the comment to reflect your changes.

Reply:
Also your changes remove the need of this [line](https://github.com/zulip/zulip/blob/03f95ba993819d74e2f6ed91123ae9648bcb712c/static/js/reactions.js#L412), please remove it. :)